### PR TITLE
[15.0][ADD] stock_picking_batch_set_quantity: New module

### PR DESCRIPTION
**Pull request by carlosdauden**
@Tecnativa
Ping @sergio-teruel @CarlosRoca13

### CODE REVIEW

**Review by CarlosRoca13 — APPROVED** [review]

File: stock_picking_batch_set_quantity/__manifest__.py [hunk 1]
```diff
@@ -0,0 +1,19 @@
+# Copyright 2023 Tecnativa - Carlos Dauden
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+{
+    "name": "Stock Picking Batch Sequence",
```
Comment (sergio-teruel) [inline stock_picking_batch_set_quantity/__manifest__.py]: Change the name

File: stock_picking_batch_set_quantity/models/stock_move.py [hunk 2]
```diff
@@ -0,0 +1,12 @@
+# Copyright 2022 Tecnativa - Carlos Dauden
```
Comment (sergio-teruel) [inline stock_picking_batch_set_quantity/models/stock_move.py]: Change copyright

**Review by sergio-teruel — CHANGES_REQUESTED** [review]

**Comment by carlosdauden** [comment]
Changes done

**Review by sergio-teruel — APPROVED** [review]

**Comment by carlosdauden** [comment]
Bot, Why don't merge instead of close it?

**Pull request merged**